curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d4626420eeecd51f241c2.md

```diff
@@ -9,6 +9,8 @@ dashedName: step-89
 
 Update the callback function to take `match`, `fn`, and `args` as parameters. It should implicitly return the result of checking whether `spreadsheetFunctions` has its own property of `fn`.
 
+To check if a property on a given object exists or not, you can use the `.hasOwnProperty()` method. `someObject.hasOwnProperty(someProperty)` returns `true` or `false` depending on if the property is found on the object or not.
+
 Remember to make `fn` lower case.
```

Comment:
The note about remember to making the `fn` lower case should come before the introduction to `hasOwnProperty`.

Also, when introducing new concepts, you should follow the convention already laid out in the curriculum.

````suggestion
Update the callback function to take `match`, `fn`, and `args` as parameters. It should implicitly return the result of checking whether `spreadsheetFunctions` has its own property of `fn`.

Remember to make `fn` lower case.

To check if a property on a given object exists or not, you can use the <dfn>hasOwnProperty()</dfn> method. 

The `hasOwnProperty()` method returns `true` or `false` depending on if the property is found on the object or not.

Here is an example of how to use the `hasOwnProperty()` method:

```js
const developerObj = {
  name: 'John',
  age: 34,
}

developerObj.hasOwnProperty('name'); // true
developerObj.hasOwnProperty('salary'); // false
```
````